### ISSUE TITLE
Formatted to wrong indent

### ISSUE THREAD

**Issue by akadateppei** [nicklockwood/SwiftFormat#1216]
I updated swiftformat 0.49.12, then caught `(indent) Indent code in accordance with the scope level.`

Until then, there was no warning in the same code like below:

```swift
timer = Timer.scheduledTimer(withTimeInterval: timeInterval, repeats: false, block: { [weak self] _ in
    guard let self = self else {
        return
    }
    for vc in self.navigationController.viewControllers.reversed() {
        if let vc = vc as? ExampleViewController {
            // some code
        } else {
            break
        }
    }
    self.invalidateTimer()
})
```

It formatted to be

```swift
timer = Timer.scheduledTimer(withTimeInterval: timeInterval, repeats: false, block: { [weak self] _ in
    guard let self = self else {
        return
    }
    for vc in self.navigationController.viewControllers.reversed() {
        if let vc = vc as? ExampleViewController {
                // some code
            } else {
                break
            }
        }
        self.invalidateTimer()
})
```

.swiftformat file

```
# rules
--disable trailingClosures, wrapMultilineStatementBraces, redundantType, redundantClosure
--stripunusedargs closure-only
```

Is this bug?

**Comment by akadateppei** [comment]
It seems like to occur in `for` inside closure.

**Comment by itaiferber** [comment]
It looks like this might be related to trivia following an open brace. I can reproduce with the following minimal definition (note the comment at the start of `ƒ() {`:

```swift
struct Foo {
    func ƒ() { // comment here
        for idx in 0 ..< 100 {
            print(idx) // ⚠️ (indent) Indent code in accordance with the scope level
        } // ⚠️ (indent) Indent code in accordance with the scope level
    }
}
```

If I remove the comment, everything works as expected:


```swift
struct Foo {
    func ƒ() {
        for idx in 0 ..< 100 {
            print(idx)
        }
    }
}
```

**Comment by krin-san** [comment]
In my case there were no in-line comments, but I have a `guard` in `for` loop. I've simplified code to this example:
```swift
class Request {
    private func processResponse() {
        performAsyncJob { [weak self] fields in
            for field in fields {
                guard true else { continue } // warning: (indent) Indent code in accordance with the scope level.
                takeAction() // warning: (indent) Indent code in accordance with the scope level.
            } // warning: (indent) Indent code in accordance with the scope level.
        }
    }
}
```

**Comment by krin-san** [comment]
Might be cause by https://github.com/nicklockwood/SwiftFormat/commit/a51a305c8fbdbc9af113bc8659bf95d838cec067 commit – that's the only one related to indentation and scopes in [0.49.11...0.49.12 diff](https://github.com/nicklockwood/SwiftFormat/compare/0.49.11...0.49.12).

**Comment by jshier** [comment]
@itaiferber's hypothesis seems correct but too narrow. I saw this triggered when I added a closure return definition: `Task { () -> [Int] in`.

**Comment by itaiferber** [comment]
@jshier Yeah, sorry, I may have used the wrong terminology. What I meant was: it seems that _any_ text between the open brace and the body of the function/closure seems to trigger this.

**Comment by jshier** [comment]
No problem, and I see @krin-san said basically the same thing. I was just thrown off by the use of `trivia`.

**Comment by nicklockwood** [comment]
Fixed in 0.49.13
